### ISSUE TITLE
Freeze up after a set amount of time in Windows, unless playing audio or something

### ISSUE THREAD

**Issue by benguild** [utmapp/UTM#4781]
**Describe the issue**  
I have a Windows 11 ARM test install and noticed that unless I leave audio playing or something (even if muted) the system will eventually freeze up.

I found a [similar issue open for QEMU](https://gitlab.com/qemu-project/qemu/-/issues/334) that suggests it may be App Nap related, but I checked Activity Monitor and when this freeze occurs it does not seem to indicate that the app is napping.

**Configuration**  
* UTM Version: 4.0.9 (68)
* macOS Version: 13.0.1 (22A400)
* Mac Chip (Intel, M1, ...): M1 Max

**Comment by LostInCompilation** [comment]
Try turning off display sleep in Windows. It caused me many troubles.

**Comment by benguild** [comment]
In the VM? It's a Mac host of course.
I've already disabled what power options were available. :(

**Comment by LostInCompilation** [comment]
Yes inside the Windows VM. The option "Turn display off after" to Never.

**Comment by benguild** [comment]
Yeah, it's already set to that unfortunately :(
